src/vs/workbench/parts/execution/common/terminal.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import fs = require('fs');
+import env = require('vs/base/common/platform');
```

Comment:
`env` -> `platform`, I think the instances where this is `env` are older ones that weren't cleaned up